ansible_collections/studyly/git_mactp/galaxy.yml

```diff
@@ -1,5 +1,5 @@
-namespace: lvrfrc87
-name: git_acp
+namespace: studyly
+name: git_mactp
 version: 2.1.0
```

Comment:
Do we want to change the version? As we added new functions, but didn't break original stuff (?), we could go to 2.2.0. If changed, don't forget to adjust other mentions of if in the docs.